doc/whats_new/v0.20.rst

```diff
@@ -441,6 +441,11 @@ Imputer
   :class:`impute.SimpleImputer`. :issue:`9726` by :user:`Kumar Ashutosh
   <thechargedneutron>`.
 
+- The ``axis`` parameter was deprecated in this version. Future (and default)
```

Comment:
I may have missed something, but it seems like the axis parameter has been removed and not deprecated.

Reply:
Another thing, it might be better to say ``The axis parameter in impute.SimpleImputer ...``. I think we need to mention ``impute.SimpleImputer`` in the entry.